packages/hw-transport-node-hid-noevents/src/TransportNodeHid.ts

```diff
@@ -31,6 +33,14 @@ const isDisconnectedError = (e) =>
  * TransportNodeHid.create().then(transport => ...)
  */
 
+// Mapping errors from https://github.com/node-hid/node-hid/blob/master/src/HID.cc
+const mapError = (e) => {
```

Comment:
i think we can totally get rid of this method because it looks like it's always going to be that remap?